src/wp-includes/class-wp-block-metadata-registry.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Block Metadata Registry
+ *
+ * @package WordPress
+ * @subpackage Blocks
+ * @since 6.X.0
+ */
+
+/**
+ * Class used for managing block metadata from various sources.
+ *
+ * @since 6.X.0
+ */
+class WP_Block_Metadata_Registry {
+
+	/**
+	 * Holds the instance of this class.
+	 *
+	 * @since 6.X.0
+	 * @var WP_Block_Metadata_Registry
+	 */
+	private static $instance;
```

Comment:
Would it be better to make this a (proper) static class, no instances? If I'm reading this right there are no benefits to instantiate it.

Reply:
Agree that there is no need to instantiate it. Changed all methods to be static in 48e2bfa19ad2c7145a9edb2029a091e823302c7d and prevented instantiation.